Assets/MixedRealityToolkit.Services/InputSimulation/Editor/InputSimulationService.cs

```diff
@@ -16,8 +16,18 @@ public class MouseDelta
         public Vector3 screenDelta = Vector3.zero;
         public Vector3 viewportDelta = Vector3.zero;
         public Vector3 worldDelta = Vector3.zero;
+
+        public void Reset()
```

Comment:
nit: add comment for api docs: resets deltas to zero

Reply:
Good catch!